tests/test_make_charge_scalar.py

```diff
@@ -0,0 +1,26 @@
+import numpy
+import pytest
+from matchms import Spectrum
+from matchms.filtering import make_charge_scalar
+
+
+@pytest.mark.parametrize("input_charge, corrected_charge", [('+1', 1),
+                                                            ('1', 1),
+                                                            (' 1 ', 1),
+                                                            ('-1', -1),
+                                                            ([-1, "stuff"], -1)])
```

Comment:
I here added cases that will be tested. @davidwhealey let me know (or add) if you have additional examples that should work with the function.